scripts/build/Dyninst/testsuite.pm

```diff
@@ -57,7 +69,7 @@ sub configure {
 			  . "-DCMAKE_INSTALL_PREFIX=$base_dir/tests "
 			  . "$args->{'cmake-args'} "
 			  . "$args->{'testsuite-cmake-args'} "
-			  . "-DDyninst_DIR=\$PWD/../dyninst/lib/cmake/Dyninst "
+			  . "-DDyninst_DIR=$dyninst_lib_dir/cmake/Dyninst "
```

Comment:
The other option would be to use `Dyninst_ROOT` that could just point to the install so we don't have to track the library directory.

```suggestion
			  . "-DDyninst_ROOT=\$PWD/../dyninst"
```